annotations/shared/src/main/scala/caseapp/Annotations.scala

```diff
@@ -17,8 +17,9 @@ object ValueDescription {
 }
 
 /** Help message for the annotated argument
+  * @messageMd skip content by case-app, it can be used to generate markdown
   */
-final case class HelpMessage(message: String) extends StaticAnnotation
+final case class HelpMessage(message: String, messageMd: Option[String] = None) extends StaticAnnotation
```

Comment:
Maybe we can type this as just `String`, like
```suggestion
final case class HelpMessage(message: String, messageMd: String = "") extends StaticAnnotation
```
so that users don't have to wrap things in `Some(…)` every time they use this field. I think the empty string is ok to mark this field as unset.